addons/mail/models/mail_thread.py

```diff
@@ -606,18 +615,21 @@ def _message_track(self, fields_iter, initial_values_dict):
                 dict((col_name, initial_values_dict[record.id][col_name])
                      for col_name in changes)
             )
+            author_id = authors[record.id].id if record.id in authors else None
```

Comment:
``authors.get(record.id)`` would do the same actually :)

Reply:
But if `record.id` isn't in `authors`, it will crash as I'll do `None.id`, right ?

Reply:
Oopsie, did not see the ".id" after the getter. Sorry for that, went too fast. Need moar coffee.